src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -11,9 +11,9 @@
 type: command
 short-summary: Log in to Azure.
 long-summary: >-
-    By default, this command logs in with a user account. CLI will try to launch a web browser to log in interactively.
-    If a web browser is not available, CLI will fall back to device code login.
-
+    By default, this command logs in with a user account.
+    Azure CLI uses Web Account Manager (WAM) on Windows, and browser-based login on Linux and macOS by default.
+    If WAM or a web browser is not available, Azure CLI will fall back to device code login.
```

Comment:
@jiasli, is "fall back" easily translated to other languages?  How about .." Azure CLI will use device code login" /or/ "Azure CLI will default to a device code login"?

Reply:
Could you explain why "fall back" should be avoided? It has been used since several years ago. Device code flow has its dedicated flag `--use-device-code`. `az login` is designed to invoke browser flow. That's why this process is a "fallback".

Reply:
"fallback" means different things in different contexts, and when a noun, it is "fallback".  When a verb or adverb, it is "fall back".  When talking about daylight savings time, you lose an hour.  When talking about someone on a ladder, they fell over backward. I'm suggesting we use a phrase that is easily translated, but I defer to engineering's judgement.